### ISSUE TITLE
Shebang is also added to files that originally did not have a shebang

### ISSUE THREAD

**Issue by GoodbyeNJN** [huozhi/rollup-preserve-directives#24]
Suppose I have the following file structure:

```
./
├── src/
│   ├── cli.ts
│   ├── client.tsx
│   └── server.tsx
└── rollup.config.js
```

```js
// rollup.config.js
import typescript from "@rollup/plugin-typescript";
import directives from "rollup-preserve-directives";

export default {
    input: ["src/cli.ts", "src/client.tsx", "src/server.tsx"],

    output: {
        dir: "dist",
        format: "es",
    },

    plugins: [typescript({ jsx: "react" }), directives()],
};
```

```ts
// cli.ts
#!/usr/bin/env node

console.log("Hello World!");
```

```tsx
// client.tsx
"use client";

export default () => "Hello World!";
```

```tsx
// server.tsx
"use server";

export default () => "Hello World!";
```

When I use rollup to bundle, the output files have their corresponding directives , which is as expected. But at the same time, every file has a shebang inserted, while in fact only cli.js needs to keep the shebang, and other files do not need to insert the shebang.

**Comment by huozhi** [comment]
Thanks for the report, I'll take a look at this multi input case 👍

**Comment by huozhi** [comment]
Fixed in v1.1.2
